test/functorch/test_vmap_registrations.py

```diff
@@ -44,7 +44,9 @@
     "aten::to.dtype_layout",
     "aten::to.other",
     "aten::upsample_bicubic2d.vec",
+    "aten::_upsample_bicubic2d_aa.vec",
```

Comment:
I see that this is consistent with batching rule for other upsampling ops but 
given that these ops are composite, I think it would make sense to decompose them instead of adding a batching rule (i.e. adding it to [BatchRulesDecomposition](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/functorch/BatchRulesDecompositions.cpp#L302)). I am curious to know if decomposing works.

If it works, we should decompose the existing ones as well.

cc: @zou3519 

Thanks!

Ref:

https://github.com/pytorch/pytorch/blob/c71a64ccce7215ab9b3938582c6249b62a72be8b/aten/src/ATen/native/native_functions.yaml#L12191-L12193

Reply:
Thanks Kshiteej, if I understand correctly the idea is to do
```
# in BatchRulesModules.cpp
EXISTING_BDIM(_upsample_bilinear2d_aa);
# and in BatchRulesDecompositions.cpp
OP_DECOMPOSE2(_upsample_bilinear2d_aa, vec);
```
however this leads to a compilation problem which comes from SymInt vs long types incompatibility: Native functions declares one arg as `SymInt[]?` but implementation uses `OptionalIntArrayRef` instead of `OptionalSymIntArrayRef`. Adapting the signature everywhere would require significant changes in this function and also underlying functions like `compute_output_size` which is also used in quantized upsampling... I propose to explore that and apply in a follow-up PR.

Reply:
I see, thanks for looking into it.

Reply:
yes, let me try to do that.